tests/local/network.go

```diff
@@ -57,6 +57,13 @@ type LocalNetwork struct {
 
 const (
 	fundedKeyStr = "56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
+	cChainConfig = `{
```

Comment:
nit: This is used for both the C-Chain and subnet chain config. Can we rename to `warpEnabledChainConfig` or something similar?